pulsar-broker-common/src/main/java/org/apache/pulsar/broker/web/RequestLogger.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.broker.web;
+
+import java.util.TimeZone;
+import org.eclipse.jetty.server.CustomRequestLog;
+import org.eclipse.jetty.server.Slf4jRequestLogWriter;
+
+/**
+ * Common class to initialize a standard request logger across all pulsar components.
+ */
+public class RequestLogger {
```

Comment:
knitting about the name of this class. The name of the class gives the impression that this is some kind of request logging abstraction. That seems misleading since this is a class for creating Jetty's RequestLog instances which will be used later for Jetty configuration. JettyRequestLogFactory would be more accurate naming. 

side note: Later on, it might be worth broadening the scope of the class to cover more of the Jetty initialization and that would impact the naming later. It seems that there's duplicated code for initializing Jetty in all different locations. That could be covered in a separate PR.

Reply:
I agree that the class name should be more specific. Thanks for the suggestion. Updated in the most recent commit.